File: backend/src/main/core/models/ActivityType.java
package main.core.models;

public final class ActivityType extends Entity {
	private String name;

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public ActivityType(String name) {
		super();
		this.setName(name);
	}
}

File: frontend/Bored_V2/models/ActivityType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bored_V2.models
{
    internal sealed class ActivityType : Entity
    {
        private string name;

        public string Name
        {
            get
            {
                return name;
            }
        }

        public ActivityType(string name)
        {
            this.name = name;
        }
    }
}
